<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Outputform</class>
 <widget class="QWidget" name="Outputform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
